packages/multi-select-combo-box/test/basic.test.js

```diff
@@ -846,6 +855,49 @@ describe('basic', () => {
       inputElement.click();
       expect(internal.opened).to.be.false;
     });
+
+    describe('dataProvider', () => {
+      let spyAsyncDataProvider;
+
+      const ITEMS = ['apple', 'banana', 'lemon', 'orange', 'pear'];
+
+      const getDataProvider = (allItems) => (params, callback) => {
+        const filteredItems = allItems.filter((item) => item.indexOf(params.filter) > -1);
+        const size = filteredItems.length;
+        const offset = params.page * params.pageSize;
+        const dataProviderItems = filteredItems.slice(offset, offset + params.pageSize);
+        callback(dataProviderItems, size);
+      };
+
+      const dataProvider = getDataProvider(ITEMS);
+
+      const asyncDataProvider = (params, callback) => {
+        setTimeout(() => dataProvider(params, callback));
+      };
+
+      beforeEach(() => {
+        comboBox.items = undefined;
+        spyAsyncDataProvider = sinon.spy(asyncDataProvider);
+        comboBox.dataProvider = spyAsyncDataProvider;
+      });
+
+      it('should not fetch items from the data-provider when readonly', async () => {
+        comboBox.opened = true;
+        // Wait for the async data provider timeout
+        await aTimeout(0);
+        expect(spyAsyncDataProvider.called).to.be.false;
+      });
+
+      it('should not update the dropdown items from the data-provider', async () => {
```

Comment:
The second test is redundant as soon as the first one passes, but it failed on master so I'd like to keep both.